src/logger/src/logger.rs

```diff
@@ -517,6 +527,16 @@ mod tests {
         );
     }
 
+    fn get_thread_name() -> String {
+        let handle = thread::current();
+        let thread_name;
+        match handle.name() {
+            Some(x) => thread_name = x.to_string(),
+            None => thread_name = "-".to_string(),
+        }
+        thread_name
```

Comment:
To make this shorter, thread_name can be removed and you could return **x.to_string()** or **"-".to_string()** directly from the match case. The same could be said about line 261. We could push the new strings directly without the need of a new variable.